dashboard/src/components/AppView/AppView.tsx

```diff
@@ -61,7 +66,7 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
     const deployments = manifest.filter(d => d.kind === "Deployment");
     const services = manifest.filter(d => d.kind === "Service");
     const apiBase = `ws://${window.location.host}/api/kube`;
-    const sockets: WebSocket[] = [];
+    const sockets = this.state.sockets;
```

Comment:
noticed we'd accidentally leak sockets when switching namespaces, so this ensures we add to a list in the state and close them all when needed